README.md

```diff
@@ -101,10 +101,10 @@ arguments and options you expect. All other commandeer constructs (described bel
 are placed under it. They are all optional - although you probably want to use
 at least one, right?
 
-**subcommand `identifier`, `name`**
+**subcommand `identifier`, `name`** or **subcommand `identifier`,`names`**
```

Comment:
Let's formulate it as such:

`**subcommand `identifier`, `name` [, alias1, alias2...]**`

because to the developer that command is the same.